scripts/jest/setupTests.js

```diff
@@ -292,9 +292,14 @@ function lazyRequireFunctionExports(moduleName) {
         // If this export is a function, return a wrapper function that lazily
         // requires the implementation from the current module cache.
         if (typeof originalModule[prop] === 'function') {
-          return function () {
+          const wrapper = function () {
             return jest.requireActual(moduleName)[prop].apply(this, arguments);
           };
+          // We use this to trick the filtering of Flight to exclude this frame.
+          Object.defineProperty(wrapper, 'name', {
+            value: '(<anonymous>)',
+          });
```

Comment:
This hack wrapper adds another stack frame which I'd rather not include in the assertions in the tests.

I can use this hack in the first phase because so far I'm just asserting on the RSC protocol but this won't work with the client where we won't filter it in React but rather rely on the UI to filter this out.